quizer.py

```diff
@@ -28,6 +36,25 @@ def welcome_page():
         if not username:
             username = session['username'] = request.form['username']
         if username:
+            if not questions_list:
+                with open('data/quiz.csv', 'rb') as csvfile:
+                    spamreader = csv.reader(csvfile, delimiter=';', quotechar='|')
+                    for row in spamreader:
+                        q = unicode(row.pop(0), "utf8")
+                        correct_answer_letter = row.pop()
+                        #correct_answer = ord(correct_answer_letter)-ord('A')
+                        answers_dict = {}
+                        for i, answer in enumerate(row):
+                            
+                            answers_dict.update({unichr(i+ord('A')):unicode(answer, "utf8")})
+                            
+                        questions_list.append({'correct':correct_answer_letter, 'answers':answers_dict, 'question':q})
+                            
+                pp.pprint(questions_list)
```

Comment:
nie zostawiaj printów w kodzie, jak tylko do debugowania to nie commituj a jak do oglądania danych to do pliku log (moduł logging)